test/ETO/PlaceholderEquityTokenController.js

```diff
@@ -494,14 +493,13 @@ contract("PlaceholderEquityTokenController", ([_, admin, company, nominee, ...in
       }
     }
     it("should allow transfer if transfers disabled only from registered ETO and only after Setup state", async () => {
-      await deployController({ ALLOW_RETAIL_INVESTORS: true, ENABLE_TRANSFERS_ON_SUCCESS: false });
+      await deployController({ ENABLE_TRANSFERS_ON_SUCCESS: false });
       await deployETO();
       await testTransfersInOffering(false);
     });
 
     it("should allow transfers after eto if requested in terms", async () => {
       await deployController({
-        ALLOW_RETAIL_INVESTORS: false,
```

Comment:
shouldn't we force ETOConstraints? do we rely on some implicit settings fixtures? we need to be sure essential parameters are set explicitely in tests

Reply:
In the current code this test is not related in any way to eto contraints, only to etoterms. so I can't really change that here, can I?

Reply:
we need to use eto constraints that allow disabling and enabling of transferability, we could for example use VMA product which has transfers disabled without possibility to turn them on. in such case test above will not pass. why not set product explicitely?